lib/control/v1/properties.js

```diff
@@ -1,13 +1,37 @@
 'use strict';
 
+const getNestedProperty = require('../../util').getNestedProperty;
+const STATUS_CODES = require('../../util/status-codes');
+
 /**
  * JSON output API
  *
  * @param {Express.App} app
  * @param {Properties} storage
  */
 exports.attach = function attach(app, storage) {
-  app.get('/v1/properties', function handler(req, res) {
+  app.get('/v1/properties/:property*', function handler(req, res, next) {
+    const properties = storage.properties;
+    const prop = req.params.property;
+
+    Log.log('INFO', 'Params: ', req.params);
+
+    if (!properties[prop]) {
+      return next(new Error(`Property ${prop} not found`));
+    }
+
+    const value = properties[prop];
+    const extra = req.params[0].split('/').filter(Boolean);
+
+    res.json(getNestedProperty(value, Array.from(extra)));
+  });
+
+  app.get('/v1/properties*', function handler(req, res) {
```

Comment:
Yeah, that's what the `/v1/properties*` route covers, although that `*` glob at the end doesn't really need to be there.